setup.cfg

```diff
@@ -45,7 +45,7 @@ dev =
     pylint >=2.11,<3 # code linter
     setuptools >=42,<68
     twine >=3.4,<5 # PyPI
-    types-pkg-resources >=0.1.3 # mypy type hints
+    types-setuptools >= 71.1.0.20240726 # mypy type hints
```

Comment:
The other package was deprecated and I could not install it anymore. So I included the successor.